src/bootstrap/config.rs

```diff
@@ -942,7 +942,7 @@ impl Config {
         }
 
         config.changelog_seen = toml.changelog_seen;
-        config.config = toml_path;
+        config.config = if toml_path.exists() { Some(toml_path) } else { None };
```

Comment:
I don't think this is quite the right check. Imagine that someone runs `x setup --config ./custom-config` when that file doesn't yet exist. I think the right thing to do there is create `./custom-config`, not `./config.toml`. The property we want is "using a default file that doesn't yet exist"; "default" there is very important. Try instead setting `config.config` on line 929, and setting it to `None` in the else branch on 931.

Reply:
Ah, I see. Thanks! I'll fix it and test